apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -367,10 +377,13 @@ public Bundle findByPatient(
 
     eobs.sort(R4ExplanationOfBenefitResourceProvider::compareByClaimIdThenClaimType);
 
+    Bundle bundle =
+        TransformerUtilsV2.createBundle(paging, eobs, loadedFilterManager.getTransactionTime());
+
     // Add bene_id to MDC logs
-    TransformerUtilsV2.logBeneIdToMdc(beneficiaryId);
+    LoggingUtils.logBenesToMdc(bundle);
 
-    return TransformerUtilsV2.createBundle(paging, eobs, loadedFilterManager.getTransactionTime());
+    return bundle;
```

Comment:
My fault for leading you down this path but this piece of code can be reverted so that we will correctly log the bene_id when it has no EOBs (at the small risk of logging fake bene_ids).

Reply:
reverted for both v1 and v2